### ISSUE TITLE
static domain or reserved ip

### ISSUE THREAD

**Issue by janlishak** [Eagles-DevOps/MiniTwit#83]
from https://github.com/itu-devops/lecture_notes/blob/master/sessions/session_03/README_TASKS.md

"
Hint: To make future operations more smooth, you might consider to use a [floating IP](https://docs.digitalocean.com/products/networking/floating-ips/) 'infront' of your actual server, see the [API spec.](https://docs.digitalocean.com/reference/api/api-reference/#tag/Floating-IPs) for receiving one.
"

**Comment by janlishak** [comment]
sources:
load balancing https://www.youtube.com/watch?v=RbpWNw6H1HQ
https://docs.digitalocean.com/glossary/ttl/
https://serverfault.com/questions/72363/what-percentage-of-nameservers-honor-ttl-these-days

the simples solution to our problem (changing of the public ip) would be to use **reserved ip**
[digital ocean reserved IP](https://docs.digitalocean.com/products/networking/reserved-ips/)

**todo**
to implement this in our terraform configuration we need to
- create reserved ip in digital ocean using the UI. (just one-time action since we'll use the same for the rest of the semester)

change terraform configuration to
1. unassigned the reserved ip from the previous droplet 
2. assigned the ip to a new droplet

In the future we might consider using digital ocean load balancer. 

I checked some source about TTL. There's like 95% that the simulator will honor the DNS 
settings to not cache the ip for longer then something like 5 minutes if we set it right.
However we cannot be certain, so for now I would not recommended based on the brief research I did.
